tcp/session.go

```diff
@@ -175,37 +175,37 @@ func (s *Session) handleBlockwise(w *ResponseWriter, r *pool.Message) {
 	s.handler(w, r)
 }
 
-func (s *Session) handleSignals(w *ResponseWriter, r *pool.Message) {
+func (s *Session) handleSignals(r *pool.Message) bool {
 	switch r.Code() {
 	case codes.CSM:
 		if s.disablePeerTCPSignalMessageCSMs {
-			return
+			return true
 		}
 		if size, err := r.GetOptionUint32(coapTCP.MaxMessageSize); err == nil {
 			atomic.StoreUint32(&s.peerMaxMessageSize, size)
 		}
 		if r.HasOption(coapTCP.BlockWiseTransfer) {
 			atomic.StoreUint32(&s.peerBlockWiseTranferEnabled, 1)
 		}
-		return
+		return true
 	case codes.Ping:
 		if r.HasOption(coapTCP.Custody) {
 			//TODO
```

Comment:
What kind of TODO needs to be done here? Please create issue for that on github.

Reply:
https://github.com/plgd-dev/go-coap/issues/11